io/src/test/resources/org/commonjava/maven/ext/manip/io/pme.yaml

```diff
@@ -0,0 +1,11 @@
+da:
+  configKey: value
+  configHierarchy:
+    configKey1: value1
+
+pme:
+  version.suffix: redhat-1
+  dependencyExclusion.com.google.guava:guava@*: 18.0
+  repo-reporting-removal: true
+  jsonUpdate: >
+    distro/data/src/main/resources/data/basic-settings.apiman.json:$$..apimanVersion:${apiman:version}\,manager/ui/war/npm-shrinkwrap.json:$$..resolved:
```

Comment:
In terms of escaping anything for yaml, we don't do anything special (currently) in PME - we delegate everything to https://bitbucket.org/asomov/snakeyaml 
I suspect there still may be some corner cases come up ....